packages/11ty/_includes/components/table-of-contents/item/grid.js

```diff
@@ -59,31 +59,41 @@ module.exports = function (eleventyConfig) {
     /**
      * Create cards for page items
      */
-    const imageAttribute = image || pageFigure || pageObject ? "image" : "no-image"
-    const slugPageAttribute = children ? "slug-page" : ""
+    const imageAttribute = image || pageFigure || pageObject ? 'image' : 'no-image'
+    const slugPageAttribute = children ? 'slug-page' : ''
 
     let imageElement
     switch (true) {
       case !!image:
-        imageElement = html`<div class="card-image">
+        imageElement = html`
+          <div class="card-image">
             <figure class="image">
               <img src="${path.join(imageDir, image)}" alt="" />
             </figure>
-          </div>`
+          </div>
+        `
         break
-      case !!pageFigure:
+      case !!pageFigure: {
         const firstFigure = firstPageFigure ? getFigure(pageFigure[0]) : null
```

Comment:
@hbalenda is `firstPageFigure` supposed to be destructured from `page.data`? linter says that's undefined